manifests/params.pp

```diff
@@ -31,6 +31,12 @@
     $service_enable = true
   }
 
+  if ($::operatingsystem == 'Ubuntu' and $::operatingsystemrelease in ['15.04','15.10','16.04','16.10']) or ($::operatingsystem == 'Debian' and $::operatingsystemrelease == '8') {
```

Comment:
It seems "Facter::Core::Execution" is not present in facter 1.7.5, which comes with Ubuntu 16.04.
I had to use the "old %x" Sysntax to make this work:

```ruby
Facter.add(:gitlab_systemd) do
  confine :kernel => :linux
  setcode do
    %x{ps -p 1 -o comm=}.gsub("\n",'') == 'systemd'
  end
end
```
Or did i miss something here?